accessibility.h

```diff
@@ -31,12 +31,38 @@
 #endif
 
 #include "configuration.h"
+#include "tasks/tasks_internal.h"
+
+#ifdef HAVE_THREADS
+#include "rthreads/rthreads.h"
+#endif
 
 typedef struct
 {
+   /* The last request task, used to prepare and send the translation */
+   retro_task_t *request_task;
+   
+   /* The last response task, used to parse costly translation data */
+   retro_task_t *response_task;
+   
+   /* Timestamp of the last translation request */
+   retro_time_t last_call;
+
+   /* Frame captured during the last call to the translation service */
+   uint8_t *last_image;
+   int last_image_size;
+
+   /* 1 if the automatic mode has been enabled, 0 otherwise */
    int ai_service_auto;
-   /* Is text-to-speech accessibility turned on? */
+   
+   /* Text-to-speech narrator override flag */
    bool enabled;
+   
+#ifdef HAVE_THREADS
```

Comment:
This is a pointer variable, so it is still misplaced here. It should go near the top of the struct.

Reply:
Ok I'll fix that